consensus/spos/worker.go

```diff
@@ -5,6 +5,7 @@ import (
 	"encoding/hex"
 	"errors"
 	"fmt"
+	"github.com/ElrondNetwork/elrond-go/redundancy"
```

Comment:
go imports. Maybe set your IDE to show these as warnings?

Reply:
done